//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcdefg_NO_AP.h
//
// Code generation for function 'abcdefg_NO_AP'
//

#ifndef ABCDEFG_NO_AP_H
#define ABCDEFG_NO_AP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcdefg_NO_AP(double P_init, double V_init, double A_init, double P_wayp,
                   double A_wayp, double V_max, double A_min, double J_max,
                   double J_min, double t[7]);

#endif
// End of code generation (abcdefg_NO_AP.h)
